scheduler/src/cook/rest/api.clj

```diff
@@ -1942,10 +1966,11 @@
   to Datomic.
   Preconditions:  The context must already have been populated with both
   ::jobs and ::groups, which specify the jobs and job groups."
-  [conn {:keys [::groups ::jobs ::pool] :as ctx}]
+  [conn {:keys [::groups ::jobs ::job-pool-name-maps] :as ctx}]
```

Comment:
This code is doing something weird. If  ::job-pool-name-maps is specified, it ignores ::jobs here and never makes transactions. Then later, we use jobs (unconditionally) when doing implicit-groups. 

If this is intentional, please document why. 

To me it seems like this should be refactored so that the caller puts puts the jobs into ::job-pool-name-maps, and ::jobs goes away.

Or if thats really hard to do, there should be a comment (and an assert) to point out that both can't be non-empty and/or catch when both of these are non-empty.

Reply:
Good call, no need for `::jobs` anymore, it's now removed.